stdlib/email/base64mime.pyi

```diff
@@ -1,9 +1,13 @@
 __all__ = ["body_decode", "body_encode", "decode", "decodestring", "header_encode", "header_length"]
 
-def header_length(bytearray: str | bytes) -> int: ...
-def header_encode(header_bytes: str | bytes, charset: str = ...) -> str: ...
-def body_encode(s: bytes, maxlinelen: int = ..., eol: str = ...) -> str: ...
-def decode(string: str | bytes) -> bytes: ...
+from _typeshed import ReadableBuffer
+
+def header_length(bytearray: bytes | bytearray) -> int: ...
```

Comment:
My thinking was that this function is about counting bytes using `len()`, and that doesn't make sense for `str` because a character can be multiple bytes. I guess it works for ASCII strings, though.

Reply:
At first I was not going to comment this for that reason, but seeing the test using `str` tipped me to comment on this.